src/globalStyles/global.scss

```diff
@@ -44,11 +44,16 @@ button:active, button:focus, div:focus, div:active {
 
 a {
   cursor:pointer;
-  text-decoration:none !important;
+  text-decoration:none;
+
+  &.disabled {
```

Comment:
annoying.  we can't use <button> because a disabled button has no mouseover events and thus can't have tooltip to explain why disabled. classic!

Reply:
since standard behavior is disabled doesnt receive mouse events, and we're potentially programming around that elsewhere, can you make this selector more specific?

Reply:
We get the benefit of the bootstrap "disabled" styles.  I think this i a better default behavior.  Not sure what the negative repercussions could be.

Reply:
it means that disabled buttons can still be clicked and potentially cause bugs whereas previously onClick could assume it wasnt disabled.

just make it more specific like `&.disabled { &.yes-pointer-events {}}`